simplesat/sat/policy/undetermined_clause_policy.py

```diff
@@ -58,6 +58,11 @@ def _build_id_to_clauses(self, clauses):
         for c in clauses:
             for l in c.lits:
                 table[abs(l)].append(c)
+
+        # Make sure all installed packages appear in the table
+        for pid in self._installed_ids:
+            table[pid]
```

Comment:
For exceptions that are related to internal consistency, we could raise a `RuntimeError`